packages/client-search/src/methods/client/updateApiKey.ts

```diff
@@ -35,17 +35,16 @@ export const updateApiKey = (base: SearchClient) => {
       'description',
       'maxQueriesPerIPPerHour',
       'maxHitsPerQuery',
-    ];
+    ] as const;
```

Comment:
just curiosity, there's no mutation on `apiKeyFields` below, and why `as const`?

Reply:
without `as const`, the type of `updatedField` lower is string, not the actual field values

Reply:
Ah I didn't know. Thanks!